ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -39,7 +84,7 @@ class SizeComparisonViewController: UIViewController {
             return
         }
 
-       removeNodeIfExistAlready()
+        removeNodeIfExistAlready()
```

Comment:
those whitespace! do a linting pass ?

Reply:
That line was not properly indented so I did a re-Indent (^I) to that line.

Reply:
I don't see that change in the last commit

Reply:
you are right! done